README.md

```diff
@@ -125,6 +125,9 @@ statistics. In addition, when `aws_dimensions` is provided, the exporter needs
 to do API requests to determine what metrics to request. This should be
 negligible compared to the requests for the metrics themselves.
 
+In the case that all `aws_dimensions` are provided in the `aws_dimension_select` list, the exporter will not perform the
+above api request, which might also reduce the cost.
```

Comment:
This would do a full cross-product, which you should mention. That potentially could be more expensive.

Reply:
@brian-brazil  I tried to explain it more hope it is more clear.